scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
+  
```

Comment:
Please don't make these kinds of changes unnecessarily it make the diff longer and has no benefit.